seed/building_sync/schemas/BuildingSync_v2_2_0.xsd

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:auc="http://buildingsync.net/schemas/bedes-auc/2019" xmlns:gbxml="http://www.gbxml.org/schema" targetNamespace="http://buildingsync.net/schemas/bedes-auc/2019" elementFormDefault="qualified" attributeFormDefault="unqualified" version="2.2.0">
-  <xs:import namespace="http://www.gbxml.org/schema" schemaLocation="http://www.gbxml.org/schema/6-01/GreenBuildingXML_Ver6.01.xsd"/>
+  <xs:import namespace="http://www.gbxml.org/schema" schemaLocation="https://github.com/BuildingSync/gbXML_Schemas/releases/download/v6.01/GreenBuildingXML_Ver6.01.xsd"/>
```

Comment:
I also tested our 2.2.0 and 2.3.0 exports, and realized it was broken. This is because the original schemas point to the "official" gbxml website which we've had issues with (see https://github.com/BuildingSync/schema/issues/393)
So I updated those two schemas to reference BuildingSync's fork of the gbxml schema

Reply:
thanks!